src/ontology/mondo-ingest.Makefile

```diff
@@ -631,7 +631,8 @@ $(SYN_SYNC_DIR)/%-synonyms.added.robot.tsv $(SYN_SYNC_DIR)/%-synonyms.confirmed.
 	--outpath-added $(SYN_SYNC_DIR)/$*.synonyms.added.robot.tsv \
 	--outpath-confirmed $(SYN_SYNC_DIR)/$*.synonyms.confirmed.robot.tsv \
 	--outpath-updated $(SYN_SYNC_DIR)/$*.synonyms.updated.robot.tsv \
-   	--outpath-combined $(TMPDIR)/synonym_sync_combined_cases_$*.tsv
+   	--outpath-combined $(TMPDIR)/synonym_sync_combined_cases_$*.tsv \
```

Comment:
Adding this line `--outpath-combined $(TMPDIR)/synonym_sync_combined_cases_$*.tsv` back in looks like an accident.

Reply:
Oh no, what that is is the addition of a ` \` at the end of that line.

The combined files are being removed in the PR I'm working on next week.

Reply:
ok, i did not realize that change was only in the other PR